bpf/lib/common.h

```diff
@@ -247,6 +261,14 @@ enum {
 #define DROP_PROXYMAP_CREATE_FAILED	-161
 #define DROP_POLICY_CIDR		-162
 
+/* Cilium metrics reason for forwarding packet */
+#define REASON_FORWARDED  0
```

Comment:
Add a comment which states that reason > 0 corresponds to -(DROP_*)

Reply:
sure

Reply:
done